src/sentry/static/sentry/app/views/projectInstall/createProject.tsx

```diff
@@ -301,7 +310,12 @@ class CreateProject extends React.Component<Props, State> {
             )}
           </HelpText>
           <PageHeading withMargins>{t('Choose a platform')}</PageHeading>
-          <PlatformPicker platform={platform} setPlatform={this.setPlatform} showOther />
+          <PlatformPicker
+            platform={platform}
+            category={category}
```

Comment:
We should probably name these something like `defaultCategory` or something, since it is not controlled.

Reply:
@EvanPurkhiser I was following the same pattern as `platform`. But maybe it makes sense to change that to `defaultPlatform` as wel.

Reply:
Uhg, I thought platform was controlled. But you're right it's not

Reply:
haha no i was wrong and you were right, gonna fix